frontend/src/aws-exports.js

```diff
@@ -18,7 +18,7 @@ const config = {
         domain: `${env?.fam_cognito_domain.value}.auth.ca-central-1.amazoncognito.com`,
         scope: ['openid'],
         redirectSignIn: `${env?.front_end_redirect_base_url.value}/authCallback`, // For some reason, vue nested path (/cognito/callback) does not work yet.
-        redirectSignOut: `${env?.front_end_redirect_base_url.value}`,
+        redirectSignOut: `${env.frontend_logout_chain_url.value}${env?.front_end_redirect_base_url.value}`,

```

Comment:
It's very minor, but let's add `?.` to the `${env` for the first part of the url.